src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -271,7 +271,10 @@ public void Initialize(string fileName)
             {
                 // failed to find the VS Service
             }
+        }
 
+        public void Initialize(string fileName)
+        {
```

Comment:
>{ [](http://example.com/codeflow?start=8&length=1)

Call `InitializeNatvisServices`?

Reply:
InitializeNatvisServices should only be called once. I think these we should add this per natvis file.